### PR TITLE
UD-1829: Update documentation to include the oauth authorization process

### PR DESCRIPTION
**Pull request by knrc**
## Description
Updated instructions to include the oauth authorization process.

## Checklist
- [X] I have labeled this PR with the relevant [Type labels](https://github.com/undistro/.github/labels?q=Type%3A)
- [ ] I have documented my code (if applicable)
- [ ] My changes are covered by tests

### CODE REVIEW

File: docs/assets/zora-dashboard-screenshot.png
Comment (lucasjct) [inline docs/assets/zora-dashboard-screenshot.png:1]: May be we can use this image selected by Arley. It is the same used on Getup site.

https://framerusercontent.com/images/cgEs9IpvuZwtRqRVHBCRQHnt8s4.png?scale-down-to=2048

File: docs/dashboard.md [hunk 1]
```diff
@@ -15,42 +15,65 @@ across all your clusters. You can also invite users to your workspace.
    <a href="https://zora-dashboard.undistro.io/" class="md-button">Try Zora Dashboard</a>
 </div>
 
-Currently, it's free for up to 3 clusters.
-Please [contact us](https://undistro.io/contact){:target="_blank"} if you need unlock more clusters in Zora Dashboard.
+Zora Dashboard offers a starter plan for 14 days, after which it will revert to the free plan which provides access for 2 clusters with up to 10 nodes per cluster.
```
Comment (lucasjct) [inline docs/dashboard.md:18]: It would be interesting to give highlight  " ... __starter plan__ for __14 days__

**Review by matheusfm — APPROVED** [review]

**Review by lucasjct — APPROVED** [review]
LGTM. I just mentioned 2 suggestions.

File: docs/assets/zora-dashboard-screenshot.png
Comment (knrc) [reply docs/assets/zora-dashboard-screenshot.png:1]: I did look at that image, unfortunately it doesn't show the button.

File: docs/dashboard.md (hunk 1 above)
Comment (knrc) [reply docs/dashboard.md]: I've made it bold and italic, check it out and see what you think

**Review by lucasjct — APPROVED** [review]
lgtm

**Pull request merged**